src/nvim/generators/gen_keysets.lua

```diff
@@ -38,7 +51,7 @@ local function sanitize(key)
   return key
 end
 
-for name, keys in pairs(keysets) do
+for name, keys in pairsByKeys(keysets) do
```

Comment:
same here. declaration order already exists, so use it. no artificial sort to paper over problems that just can be eliminated.

Reply:
Hmm, I thought `ipairs` only works with numeric keys? I'm getting an error at least when I use it. Or did you mean I should use something else?

Reply:
I mean definition order, like in my first comment, not the second one. sorry for the _syftningsfel_ :)